Lombiq.Hosting.Tenants.Management.Tests.UI/Extensions/TestCaseUITestContextExtensions.cs

```diff
@@ -19,10 +19,18 @@ public static async Task TestShellSettingsEditorFeatureAsync(this UITestContext
         await context.FillInEditorThenCheckValueAsync(
             "{\"TestKey\":{\"TestSubKey\":{\"TestSubOptions\":{\"Test\": \"TestValue\"}}}}",
             "TestValue");
+
+        await context.FillInEditorThenCheckValueAsync(
+            "{\"TestKey\":{\"TestSubKey\":{\"TestSubOptions\":{\"NewKey\": \"NewValue\"}}}}",
+            "NewValue");
```

Comment:
Due to how `FillInEditorThenCheckValueAsync()` only checks for `"TestKey.TestSubKey.TestSubOptions.Test"`, this will only pass because `SelectToken()` will return null. It's not actually checking the value.

Reply:
By checking the value do you want to access ShelSettings from here?

Reply:
ahh nvm, I got it.